integration_test/sql/migration.exs

```diff
@@ -111,6 +111,29 @@ defmodule Ecto.Integration.MigrationTest do
     end
   end
 
+  defmodule RenameColumnMigration do
+    use Ecto.Migration
+
+    def up do
+      create table(:rename_col_migration) do
+        add :to_be_renamed_inside_alter, :integer
+        add :to_be_renamed_outside_alter, :integer
+      end
+
+      alter table(:rename_col_migration) do
```

Comment:
Can you please add another operation inside the alter table? You can add any column you want. The reason for this is that some operations can only exist by themselves inside `alter table`, so we always need to add another operation to help us catch those cases.